AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/AWSDataStorePlugin.swift

```diff
@@ -55,9 +55,10 @@ final public class AWSDataStorePlugin: DataStoreCategoryPlugin {
         }
     }
 
+    #if os(watchOS)
     /// No-argument init that uses defaults for all providers
     public init(modelRegistration: AmplifyModelRegistration,
-                configuration dataStoreConfiguration: DataStoreConfiguration = .default) {
+                configuration dataStoreConfiguration: DataStoreConfiguration) {
```

Comment:
removing `.default` for watchOS means we will explicitly force customers to set the configuration, which requires explicitly setting the `disableSubscriptions` parameter

Reply:
I think we talked about it, but this would technically be a breaking change for watchOS 😕